docs/Installation.md

```diff
@@ -514,14 +518,15 @@ psql -U pgsql -f ./initial-postgres.sql template1
 
 ### 3.3 Service startup (FreeBSD)
 
->
-> At this time there is no instruction for automatically running Zonemaster Web backends
-> nor Workers as services on FreeBSD.
->
+Install service scripts and start services:
 
 ```sh
-starman --error-log="$HOME/logs/error.log" --pid-file="$HOME/logs/starman.pid" --listen=127.0.0.1:5000 --daemonize /usr/local/bin/zonemaster_backend_rpcapi.psgi 
-zonemaster_backend_testagent start
+install -m 775 -g zonemaster -d /var/log/zonemaster
+install -m 775 -g zonemaster -d /var/run/zonemaster
+install -m 755 ./zm_rpcapi /usr/local/etc/rc.d/
+install -m 755 ./zm_testagent /usr/local/etc/rc.d/
+echo 'zm_rpcapi_enable="YES"' | tee -a /etc/rc.conf
+echo 'zm_testagent_enable="YES"' | tee -a /etc/rc.conf
 ```

```

Comment:
I suggest the following lines instead as it is usually written i FreeBDS instructions (e.g. FreeBSD Handbook):

Add the following lines to /etc/rc.conf:
```
zm_rpcapi_enable="YES"
zm_testagent_enable="YES"
```
Then start the daemons:
```
service  zm_rpcapi start
service zm_testagent start
```

Reply:
The `echo | tee -a` lines have the advantage that they are cut-and-pasteable, and I reckon we'll be adding those lines a lot.

Reply:
The `echo | tee -a` line assumes that rc.conf ends with a line ending. It also adds that line even if not needed (already there). Here is a new version:

```
Run the following command turn the daemons on in /etc/rc.conf:

sysrc zm_rpcapi_enable="YES"
sysrc zm_testagent_enable="YES"
```

Reply:
Neat! That's much better.